### PR TITLE
Sign up 🎉 

### PR DESCRIPTION
**Pull request by chancefeick**
Add links to sign up and pricing page. Remove early access form. Re-arrange header and footer links. Requires pricing page introduced in #24.

Screen Shot 2021-03-17 at 5 15 40 PM

### CODE REVIEW

**Comment by chancefeick** [comment]
[Includes links to pricing and signup pages in orbit website](https://trello.com/c/QymNiT1j/489-includes-links-to-pricing-and-signup-pages-in-orbit-website)

**Comment by sorich87** [comment]
Looks nice @chancefeick ! There's some parts that would cause a conflict though, as the early access form is being converted to a contact form in the #24. I'll comment on the specifics.

File: src/layouts/base.njk [hunk 1]
```diff
@@ -18,14 +18,10 @@
...
       role="document"
       class="min-h-screen flex flex-col"
-      x-data="{ showModal: false }"
-      @keydown.escape="showModal = false"
-      @togglemodal="showModal = $event.detail.value"
     >
       <main class="flex-grow w-full">
         {{ content | safe }}
       </main>
-      {% include "early-access-form.njk" %}
```
Comment (sorich87) [inline src/layouts/base.njk:28]: All the lines removed in this file are needed by #24.

File: src/includes/early-access-form.njk [hunk 2]
```diff
@@ -1,151 +0,0 @@
...
-          this.message = "Thanks! We’ll be in touch soon."
-        } catch (err) {
-          console.error(err);
-          this.submitting = false;
-          this.message = "There was an error while subscribing."
-        }
-      }
-    }
-  }
-</script>
```
Comment (sorich87) [inline src/includes/early-access-form.njk:151]: This file shouldn't be removed as it's renamed and reused in #24.

**Review by sorich87 — CHANGES_REQUESTED** [review]

**Comment by chancefeick** [comment]
> There's some parts that would cause a conflict though, as the early access form is being converted to a contact form in the #24. I'll comment on the specifics.

@sorich87 Thanks for the heads up. I'll hold off on addressing any merge conflicts here until #24 is merged first 👍🏻

**Comment by phacks** [comment]
@chancefeick Pricing page is merged!

**Comment by chancefeick** [comment]
@sorich87 Thanks for helping resolve the conflict! 🙏🏻

**Comment by phacks** [comment]
Took a tour of the preview deployment. All looks good! If we can sneak in a small improvement before deploy I noticed that the Sign Up link has no :focus styles, which makes it hard for keyboard users to know if it’s currently focused. I think the other Get Started buttons have one, so it might be a matter of applying the same classes

**Comment by chancefeick** [comment]
> I noticed that the Sign Up link has no :focus styles, which makes it hard for keyboard users to know if it’s currently focused

@phacks Great suggestion, added in https://github.com/orbit-love/orbit-web/pull/25/commits/ebf1fe36b34d01119b7ca6f28ad040cbcc4d4258.

**Review by sorich87 — APPROVED** [review]

**Comment by sorich87** [comment]
Great work Chance! Everything looks good.

**Review by phacks — APPROVED** [review]

**Pull request merged**